### PR TITLE
Quiz page

### PR DESCRIPTION
**Pull request by sumitk-singh**
### Need to review in the following
- quiz card compo
- quiz page
- API

### CODE REVIEW

File: src/API/DataComputerScienceQuiz.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import GameOver from '../Pages/quiz-page/GameOver';
+// import "../../quiz-page/Quiz.css";
+
```
Comment (Shubhamkr2610) [inline src/API/DataComputerScienceQuiz.js]: you can remove it if you are not using it

**Review by Shubhamkr2610 — APPROVED** [review]
I can see that in some of files you are importing css that you are not using it, and you have commented that , so you can remove it . 
else everything looks good to me. You can merge it
Appreciation: code is clean , readable and easy to understand

File: src/components/QuizCard/QuizCard.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  height: 1rem max-content;
+  
+  width: 20rem;
+  border-radius: 12px;
+  background: linear-gradient(var(--play-btn), white);
+
+}
+
+.btn-play {
+  border-radius: 12px;
```
Comment (tonymike99) [inline src/components/QuizCard/QuizCard.css]: You can use rems instead of px

File: src/components/QuizCard/QuizCard.css [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+#quiz-card-3 {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  border: 1px solid var(--play-btn);
+  padding: 1rem;
+  height: 1rem max-content;
+  
+  width: 20rem;
+  border-radius: 12px;
```
Comment (tonymike99) [inline src/components/QuizCard/QuizCard.css]: Use rem instead of px

**Review by tonymike99 — APPROVED** [review]
Hi Sumit,
Overall the code looks fine and well structured.
Suggestion: I have pointed out places where you can change px to rems at certain places. Avoid px for larger numbers because it is not responsive
Good to merge.

File: src/API/DataComputerScienceQuiz.js (hunk 1 above)
Comment (sumitk-singh) [reply src/API/DataComputerScienceQuiz.js]: ok

**Pull request merged**